modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/RubyClientCodegen.java

```diff
@@ -258,6 +260,11 @@ public void processOpts() {
         //writeOptional(outputFolder, new SupportingFile("base_object_spec.mustache", specFolder, "base_object_spec.rb"));
     }
 
+    @Override
+    public String toModelImport(String name) {
+        return this.toModelFilename(name);
+    }
+
```

Comment:
Fixes the import statement to use the file name rather than the model name as that is what Ruby uses to import the additional class